Sources/ProjectSpec/Scheme.swift

```diff
@@ -294,6 +306,7 @@ extension Scheme.Test: JSONEncodable {
     public func toJSONValue() -> Any {
         return [
             "gatherCoverageData": gatherCoverageData,
+            "disableMainThreadChecker": disableMainThreadChecker,
```

Comment:
It looks like `gatherCoverageData` accidentally missed out on this before, so it was easy to do, but these values should only be written if they differ from the default

Reply:
Also is `Scheme.Run` writing this property?